File: Drivers/FSCC Drivers/windows/lib/fscc/c/examples/append-status.c
#include <fscc.h> /* fscc_* */

int main(void)
{
	fscc_handle h;
    unsigned status;

	fscc_connect(0, &h);

    fscc_get_append_status(h, &status);

	fscc_enable_append_status(h);
	fscc_disable_append_status(h);

	fscc_disconnect(h);

	return 0;
}

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/append-status.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);

        var status = p.AppendStatus;
        
        p.AppendStatus = true;
        p.AppendStatus = false;

        return 0;
    }
}